subsys/mpsl/Kconfig

```diff
@@ -25,6 +25,107 @@ config MPSL_TIMESLOT_SESSION_COUNT
 	help
 	  Maximum number of timeslot sessions.
 
+DT_COMPAT_NORDIC_NRF21540_GPIO := nordic,nrf21540-fem
```

Comment:
I would consider moving the FEM configuration to a separate file: Kconfig.fem

Reply:
and maybe even consider if the settings specifically related to nRF21540 and sky66112 should be placed independent files as well.

Also, why are pin numbers defined in Kconfig and not in DTS ?

Reply:
I made the split as suggested.

In my opinion, it is not necessary to separate nRF21540 and sky66112 now. I will consider this in the future when adding nRF21540-SPI and nRF21540-mixed. Then these files will probably be too extensive.

Regarding the pin numbers, @tejlmand, you probably meant that PPI and GPIOTE are defined in Kconfig.
I will refer to the explanations @hubertmis has posted [here](https://github.com/nrfconnect/sdk-nrf/pull/2896#discussion_r537591023) - we intend to keep current proposal temporarily.